pypeit/core/telluric.py

```diff
@@ -1778,6 +1793,10 @@ class Telluric(object):
             Argument for scipy.optimize.differential_evolution which will  display status messages to the screen
             indicating the status of the optimization. See above for a description of the output and how to know
             if things are working well.
+        sensfunc (bool): default=False
```

Comment:
optional

Reply:
here and throughout this doc string. 
and replace `ndarray` with its proper link.

Reply:
Done.